test/README.md

```diff
@@ -45,6 +45,7 @@ The end-to-end tests take several parameters, which they expect to find in a fil
 | `tempFilterFuncImage` | * | Azure temperature filter function to be used. Required when running the test 'TempFilterFunc', ignored otherwise.|
 | `tempFilterImage` | * | Docker image to pull/use for the temperature filter module. Required when running the test 'TempFilter', ignored otherwise.|
 | `tempSensorImage` || Docker image to pull/use for the temperature sensor module (see the test 'TempSensor'). If not given, `mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0` is used.|
+| `edgeAgentBootstrapImage` || Docker image to pull/use for the initial startup of edgeAgent. This is a temporary parameter - only here now because the 1.0 version is incompatible with master's edgeAgent. If you want to specify your own, the container registry used is the first registry in the list of registries given by the 'registries' parameter|'
```

Comment:
I'd make it clear this is the Edge Agent image used in config.yaml, as opposed to the image given to deployments.